Protactor/specs/jankiness.js

```diff
@@ -1,8 +1,13 @@
 const assert = require('assert')
 
 describe('should test the jankiness', () => {
+    beforeAll(() => {
+        browser.waitForAngularEnabled(false)
+    })
+
     it('tests jankiness with non optimized behavior', async () => {
         browser.get('https://googlechrome.github.io/devtools-samples/jank/')
+        await browser.sleep(1000)
```

Comment:
I wonder if we can use an explicit wait here. Also can you add an `await` before the `browser.get`?